appvalidator/specs/webapps.py

```diff
@@ -267,6 +306,23 @@ class WebappSpec(Spec):
                     "navigation": {"expected_type": bool},
                 }
             },
+            # For now, we only support languages pack targeting gaia apps and
+            # only with 2.2 version.
+            "languages-target": {
+                "expected_type": dict,
+                "process": lambda s: s.process_languages_target,
+                "allowed_nodes": ["app://*.gaiamobile.org/manifest.webapp"],
+                "unknown_node_level": "error",
+                "not_empty": True,
+                "child_nodes": {
+                    "app://*.gaiamobile.org/manifest.webapp": {
+                        "expected_type": types.StringTypes,
+                        "not_empty": True,
+                        "value_matches": r"^2[.]2$"
```

Comment:
What's the reasoning for using `[.]` instead of `\.`? `[.]` seems like it should match anything to me but after some testing it only matches `.`.

Will this need to be updated when 2.3 or 2.4 is released (i'm assuming this is matching an FxOS version)?